Symfony/CS/Tests/Tokenizer/Transformer/DynamicVarBraceTest.php

```diff
@@ -26,12 +26,12 @@ public function testProcess($source, array $expectedTokens = array())
     {
         $tokens = Tokens::fromCode($source);
 
-        $this->assertSame(
-            count($expectedTokens),
+        $this->assertCount(
```

Comment:
we're not making an assertion on the thing we already have defined as correct...